extension/src/popup/components/Tabs/index.tsx

```diff
@@ -1,33 +0,0 @@
-import React, { ReactNode } from "react";
-import classnames from "classnames";
-
-import "./styles.scss";
-
-interface TabsProps {
-  tabs: string[];
-  renderTab: (tab: string) => ReactNode;
-}
-
-export const Tabs = (props: TabsProps) => {
```

Comment:
this was an unused file with similar naming to the Tokens/Collectibles tab. Removing for clarity